### ISSUE TITLE
BCH balance 0.00 without manual address scan

### ISSUE THREAD

**Issue by meinharrd** [bitpay/copay#6797]
### Issue

BCH balance remains 0.00 after wallet duplication from BTC. A manual address scan corrects the balance, but it's hard to figure out even for an advanced user that this step is needed. This feels like a bug.

### Reproduce

1. Go to "Settings" > "Enable Support Bitcoin Cash" > Switch on "Support Bitcoin Cash"
2. Press "Scan your wallets for Bitcoin Cash"
3. Press "Duplicate for BCH" on the wallet that should contain BCH
4. New wallet created by duplication will display 0.00 balance, even if the wallet contains pre-1-August BCH

### Work-around / fix

1. Go to "Settings" > "Wallets & Integrations" and select newly created BCH wallet
2. Select "More Options" > "Wallet Addresses" > "Scan addresses for funds"
3. Scanning completes, correct balance will be displayed (phew!) 😅

### Suggested solution

* Automatically initiate an address scan after "Duplicate for BCH" is pressed

### Environment

* Tested on v3.8.2 iOS and Windows

**Comment by JDonadio** [comment]
You are right, that should be fixed for the next release. Thank you very much for your report and suggestion.

**Comment by avibrazil** [comment]
I have this problem and workaround works for me.

**Comment by wansei** [comment]
Hi, I seem to have a similirar issue. I've retrieved my Mycelium wallet (no BTC in the wallet, only BCH) via your online recovery tool, which worked fine. Transaction went through, appeared in my bitpay wallet, status - receiving. Next day I checked again - still receiving (with receiving symbol next to the wallet name).  Somehow I managed to get to a page showing 1. Funds (showing correct amount) 2. Spendable funds (showing zero). After that the receiving symbol disappeared and the balance showed up correctly in the home tab. But transaction itself was in receiving status still. Only after I cleared cache through settings, both showed correctly. Please let me know if this helps you, seems buggy to me

**Comment by JDonadio** [comment]
@wansei please for your problem create a new issue.

**Comment by igycb** [comment]
Same issue of showing zero balance after using the duplicate for BCH function -- but workaround of manually scanning wallet addresses still results in 0 balance for me.

**Comment by graburn** [comment]
I had this same issue and followed the steps above. However, the wallet has been stuck with the status of "Scanning funds... please wait" for days now. Seems like it probably shouldn't take this long.

**Comment by igycb** [comment]
@JDonadio any thoughts on the issue I mentioned above? Appreciate the support.

**Comment by JDonadio** [comment]
Try with the recovery tool please: https://bitpay.github.io/copay-recovery/

**Comment by graburn** [comment]
Recovery tool had no affect with me. Wallet is still stuck at "Scanning Funds Please Wait."

**Comment by graburn** [comment]
Months later, still "Scanning funds." :(

**Comment by pietjebelkees** [comment]
I've sent bch to my btc copay wallet, when I tap settings > bitcoin cash it says that my bch are there, the exact amount. When I tab on duplicate a bch wallet is created but no sign of the bch, I already tried the recovery tool. It says 0 bch. I already checked the transaction, it has been confirmed 2 days ago.

Copay isn't doing anything about this? Many people lost money.

I even sent another 0.1 BCH as a test and even that didn't showed up, this is clearly a bug
